homeassistant/components/econet/__init__.py

```diff
@@ -1 +1,194 @@
-"""The econet component."""
+"""Support for EcoNet products."""
+import asyncio
+from datetime import timedelta
+import logging
+
+from pyeconet import EcoNetApiInterface
+from pyeconet.equipment import EquipmentType
+from pyeconet.errors import InvalidCredentialsError, PyeconetError
+import voluptuous as vol
+
+from homeassistant.config_entries import SOURCE_IMPORT
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD, TEMP_FAHRENHEIT
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.entity import Entity
+from homeassistant.helpers.event import async_track_time_interval
+
+from .const import API_CLIENT, DOMAIN, EQUIPMENT
+
+_LOGGER = logging.getLogger(__name__)
+
+ATTR_ON_VACATION = "on_vacation"
+ATTR_IN_USE = "in_use"
+ATTR_WIFI_SIGNAL = "wifi_signal_strength"
+ATTR_ALERT_COUNT = "alert_count"
+
+CONFIG_SCHEMA = vol.Schema(
```

Comment:
Config yaml is legacy for device integrating integrations. Just import the platform config to the config flow instead.

Reply:
done